.gitignore.example

```diff
@@ -0,0 +1,38 @@
+/index.php
+/assets/addons/
+/assets/core/
+/media/
+!/media/.redaxo
+/redaxo/index.php
+/redaxo/cache/
+/redaxo/src/
+/redaxo/data/core/system.log
+/redaxo/data/core/config.yml
+*.log
+/.htaccess
+
+# Addon Backup
+/redaxo/data/addons/backup/
+
+# Addon Installer: archivierte Addon-ZIPs
+/redaxo/data/addons/install/*
+
+# Addon Cronjob: Logs und Export-Dateien
+redaxo/data/addons/cronjob/cronjob.log
+*.cronjob.sql
```

Comment:
fehlt hier der pfad vorne drann?

Reply:
Wo?

Reply:
Willst du wirklich das dateipattwrn .cronjob.sql in allen pfaden von redaxo ignorieren?

Reply:
Ah, das war ein Flüchtigkeitsfehler. Wird korrigiert